cmd/csaf_checker/processor.go

```diff
@@ -677,9 +677,20 @@ func (p *processor) processROLIEFeed(feed string) error {
 		return err
 	}
 	if len(errors) > 0 {
-		p.badProviderMetadata.error("%s: Validating against JSON schema failed:", feed)
-		for _, msg := range errors {
-			p.badProviderMetadata.error(strings.ReplaceAll(msg, `%`, `%%`))
+		lengthError := false
+		for _, lerr := range errors {
+			if lerr == "/feed/entry: minimum 1 items required, but found 0 items" {
+				p.badProviderMetadata.warn(lerr)
+				lengthError = true
+			}
+		}
+		if !(lengthError && len(errors) == 2) {
```

Comment:
Checking for a length of 2 feels oddly specific here.

Reply:
If there is an error due to the length of entry, then another general error will be thrown to indicate failure of validation against the schema. So if there is a length error and *only* the general error, then there is no need to call validation an actual failure. Three or more errors would indicate additional failures.

Reply:
Part has been rewritten